src/gui/screens/documentation/actions/actions.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import SaveButton from '../../../components/button'
+
+const Actions = ({ onSave }) => {
+  return (
+    <div>
+      <SaveButton label="Save" onClick={onSave} />
+    </div>
+  )
+}
+
+Actions.propTypes = {
+  onSave: PropTypes.func.isRequired,
+}
+
+export default Actions
```

Comment:
I think this a good practice to always use `onlyUpdateForPropTypes` from `recompose`

https://github.com/Zenika/react-workbench/blob/master/src/gui/screens/workbench/state/state.jsx#L32